lib/chat_transcript_service.rb

```diff
@@ -24,12 +24,14 @@ class ChatTranscriptBBCode
 
     def initialize(
       channel: nil,
+      acting_user: nil,
```

Comment:
I don't think you wanted to have all the acting_user stuff in this PR

Reply:
@jjaffeux yes I do...how else can I pass the user to the `.title(user)` method on the channel?